pam/write/matsim.py

```diff
@@ -16,15 +17,15 @@
 from pam.utils import DEFAULT_GZIP_COMPRESSION, create_crs_attribute, create_local_dir, is_gzip
 from pam.utils import datetime_to_matsim_time as dttm
 from pam.utils import timedelta_to_matsim_time as tdtm
-from pam.vehicle import ElectricVehicle, Vehicle, VehicleType
 
 
 def write_matsim(
-    population: Population,
-    plans_path: str,
-    attributes_path: Optional[str] = None,
-    vehicles_dir: Optional[str] = None,
-    version: Optional[int] = None,
+    population,
+    plans_path: Union[Path, str],
+    attributes_path: Union[Path, str, None] = None,
```

Comment:
```suggestion
    attributes_path: Optional[Union[Path, str]] = None,
```
Although the result is the same, this is clearer as a hint, saying that useful inputs are `Path`/`str` and `None` is a placeholder to indicate you want to ignore that arg.

Reply:
Same with the below args